docs/tech_details/api/fileactionsmenu.rst

```diff
@@ -25,12 +25,13 @@ Complete list of params (including optional):
 
 	{
 		"name": "unique_name_of_file_actions_menu",
-		"displayName": "Display name (for UI listing)",
-		"actionHandler": "/action_handler_route"
+		"display_name": "Display name (for UI listing)",
+		"action_handler": "/action_handler_route"
 		"mime": "mime of files where to display action menu",
 		"icon": "img/icon.svg",
 		"permissions": "permissions",
 		"order": "order_in_file_actions_menu",
+		"version": "1.0"
```

Comment:
why need this? both versions already has different urls

Reply:
Will update docs soon